app/javascript/packs/diagnosis/content_form.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div id="content-form">
+        <div class="ui dimmable segment shadow-less" v-bind:class="{ dimmed: disabled }">
+            <div class="ui simple inverted dimmer">
+                <div class="ui text loader">{{ pageData.locales.save_loader }}</div>
+            </div>
+            <div class="ui form">
+                <div class="field">
+                    <label>{{ pageData.locales.description }}</label>
+                    <textarea rows="2" v-model="diagnosis.content"></textarea>
+                </div>
+                <button class="small ui button" v-on:click="saveButtonClicked">
+                    {{ pageData.locales.save_description }}
+                </button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import axios from 'axios'
+
+    export default {
+        name: 'content-form',
+        data () {
+            return {
+                diagnosis: { content: ''},
+                disabled: true,
+                pageData: {
+                    diagnosis_id: '',
+                    locales: {
+                        description: '',
+                        save_description: '',
+                        save_loader: ''
+                    }
+                }
+            }
+        },
+        mounted: function () {
+            this.mountPageData();
+        },
+        watch: {
+            pageData: function (val) {
+                console.log('diagnosis_id: ' + this.pageData.diagnosis_id );
+                if(val.diagnosis_id) {
+                    this.getDiagnosis();
+                }
+            }
+        },
+        methods: {
+            getDiagnosis: function () {
+                var that;
+                that = this;
+                console.log('getDiagnosis');
+                axios.get('/api/diagnoses/' + this.pageData.diagnosis_id  + '.json')
```

Comment:
Prefer using Rails-generated URLs (`diagnosis_api_path(id)`), readable by JS thanks to data attributes, such as here: https://github.com/sgmap/reso/blob/master/app/assets/javascripts/assistance.coffee